EventFilter/Utilities/plugins/RecoEventOutputModuleForFU.h

```diff
@@ -72,6 +72,7 @@ namespace evf {
 
   template<typename Consumer>
   RecoEventOutputModuleForFU<Consumer>::RecoEventOutputModuleForFU(edm::ParameterSet const& ps) :
+    edm::one::OutputModuleBase::OutputModuleBase(ps),
```

Comment:
Yes, it seems that compiler doesn't check constructor definitions in the chain (which can be in .cc files or even in a different library) and assumes default one.